### DISCUSSION TITLE
收国外邮件延迟很大

### DISCUSSION THREAD

**Discussion by udbyygyt** [Mailu/Mailu#3208] [User support]
在国内收国外邮件延迟很大，正常都慢10几分钟
我记得之前在哪里查到可以去谷歌设置域名，后面那段时间都正常没延迟了

现在又开始收件延迟了，忘记之前是怎么设置的了，有人知道吗？

**Comment by ryuushihou** [comment]
请问 你知道如何添加邮件头部吗
header_checks
